### PR TITLE
overall edits

### PR DESCRIPTION
**Pull request by graciellehigino**
PR to merge overall edits from intro, discussion, results and figure legends (ignore the name of the branch for now! :joy: )

### CODE REVIEW

**Comment by FrancisBanville** [comment]
Thanks @graciellehigino! I think your changes make the manuscript and our goals more clear. I especially like your paragraph on GBIF dates.

I think the expression "areas of uncertainty" is a bit confusing, though. Perhaps we could replace it with "areas of data deficits" or something similar? Thinking about that, I can see how this manuscript fits with our project on identifying sampling locations. It could be interesting to mention it in the discussion. 

Also, I am unsure if we should use the word "fitted" to describe the ranges that consider interaction data. We don't really fit anything statistically, so this could also be a bit confusing.

**Comment by graciellehigino** [comment]
Thanks @FrancisBanville!

I've made some changes replacing the expressions you've mentioned, next I'll add a sentence or two about our BH&R project =D

**Comment by graciellehigino** [comment]
Closes #97

File: manuscript.md [hunk 1]
```diff
@@ -448,7 +448,14 @@ accounting for these errors can improve model outputs by diminishing the error
...
 account for ecological interactions in habitat suitability models without making
 the models more complex, but by making sure (not assuming) that the input data -
-the species occurrence - actually accounts for ecological interactions. It is
+the species occurrence - actually accounts for ecological interactions. Another
+application of this method is mapping areas where data are deficient and
+helping to indicate priority sampling locations for interaction data, which can, in
+turn, reduce uncertainty in network prediction. For example, if a certain pixel
+confirms the presence of a species both with IUCN and GBIF data, but lacks
+connection between species, this pixel has a high potential to hide an
+unobserved interaction and should therefore be a priority sample location.
```
Comment (graciellehigino) [inline manuscript.md:457]: @gabrieldansereau is there a reference for our [BH&R](https://github.com/PoisotLab/BlackHolesAndRevelations) project? Should we add the repo or maybe something from your thesis?

**Comment by graciellehigino** [comment]
Merging for now to work on the response letter - gotta get the line numbers.

**Pull request merged**